packages/expo-web-browser/android/src/test/java/expo/modules/webbrowser/TestUtils.kt

```diff
@@ -0,0 +1,57 @@
+package expo.modules.webbrowser
+
+import android.content.Intent
+import io.mockk.CapturingSlot
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.just
+import org.unimodules.core.arguments.ReadableArguments
+import org.unimodules.test.core.mockkInternalModule
+import org.unimodules.test.core.readableArgumentsOf
+
+fun mockkCustomTabsActivitiesHelper(
+  services: ArrayList<String> = ArrayList(),
+  activities: ArrayList<String> = ArrayList(),
+  preferredActivity: String? = null,
+  defaultActivity: String? = null,
+  startIntentSlot: CapturingSlot<Intent>? = null,
+  defaultCanResolveIntent: Boolean = false):
+  CustomTabsActivitiesHelper {
```

Comment:
What's the correct formatting here? 🤔
Maybe we can somehow adjust kotlin formatted to behave more js/ts-style 🤔?:
```suggestion
fun mockkCustomTabsActivitiesHelper(
  services: ArrayList<String> = ArrayList(),
  activities: ArrayList<String> = ArrayList(),
  preferredActivity: String? = null,
  defaultActivity: String? = null,
  startIntentSlot: CapturingSlot<Intent>? = null,
  defaultCanResolveIntent: Boolean = false
): CustomTabsActivitiesHelper {
```

Reply:
Looks like AS formatter with my settings does not really care, and consider both correct. Will change to suggested.